### PR TITLE
GEOPY-1789: Add check for straight line breaking the triangulation for horizon refinement

### PR DESCRIPTION
**Pull request by benk-mira**
**GEOPY-1789 - Add check for straight line breaking the triangulation for horizon refinement**

### CODE REVIEW

File: octree_creation_app/utils.py [hunk 1]
```diff
@@ -13,12 +13,77 @@
...
+def surface_strip(
+    points: ObjectBase, width: float, name: str = "Surface strip"
+) -> Points:
+    """
+    Duplicate and offset co-linear input points to create a co-planar strip.
+
+    :param points: Points object whose locations are all co-linear.
+    :param width: Width used to displace existing points to create the
+        strip.  The surrounding strip will be 2*width wider and longer than
+        the input points.
```
Comment (domfournier) [inline octree_creation_app/utils.py:53]:
```suggestion
        the input points.
    :param name: Name of the new Points objects.
```

**Review by domfournier — DISMISSED** [review]
Looks great man.
Small edit to the docstring below.

File: docs/source/conf.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
 project = "octree-creation"
-release = "0.2.0-alpha.1"
+release = "0.3.0-alpha.1"
```
Comment (domfournier) [inline docs/source/conf.py:10]: Oops. Revert this.

Comment (benk-mira) [reply docs/source/conf.py]: done

File: pyproject.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "octree-creation-app"
-version = "0.2.0-alpha.1"
+version = "0.3.0-alpha.1"
```
Comment (domfournier) [inline pyproject.toml:3]:
```suggestion
version = "0.2.0-alpha.1"
```

File: octree_creation_app/__init__.py [hunk 4]
```diff
@@ -12,7 +12,7 @@
 from geoapps_utils.utils.importing import assets_path as assets_path_impl
 
 
-__version__ = "0.2.0-alpha.1"
+__version__ = "0.3.0-alpha.1"
```
Comment (domfournier) [inline octree_creation_app/__init__.py:15]:
```suggestion
__version__ = "0.2.0-alpha.1"
```

**Pull request merged**